modules/rubiconBidAdapter.js

```diff
@@ -518,7 +520,12 @@ function parseSizes(bid) {
   }
 
   // deprecated: temp legacy support
-  let sizes = Array.isArray(params.sizes) ? params.sizes : mapSizes(bid.sizes)
+  let sizes = [];
+  if (typeof utils.deepAccess(bid, 'mediaTypes.banner.sizes') !== 'undefined') {
+    sizes = mapSizes(bid.mediaTypes.banner.sizes);
+  } else {
+    sizes = Array.isArray(params.sizes) ? params.sizes : mapSizes(bid.sizes)
+  }
```

Comment:
If `params.sizes` is present it should be used over both `bid.sizes` and `mediaTypes.banner.sizes`, so this logic needs to be adjusted.